content-cache/charmcraft.yaml

```diff
@@ -11,6 +11,41 @@ platforms:
     build-for: amd64
 
 parts:
+  nginx-module-lua-upstream:
+    plugin: dump
+    source: lua-upstream-nginx-module-packaging
+    build-packages:
+      - git
+      - dpkg-dev
+      - libnginx-mod-http-ndk-dev
+      - nginx-dev
+      - liblua5.4-dev
+      - dh-lua
+    override-build: |
+      set -eux
+
+      # Fetch the module to compile
+      repo=lua-upstream-nginx-module
+      rm -Rf $repo
+      git clone https://github.com/openresty/$repo
+      cd $repo
+      git checkout 07b11be7a2e6e1b75209745915a84d281737d4d7
+
+      # Build package
+      cp -R ../debian .
+      curl https://raw.githubusercontent.com/openresty/lua-nginx-module/f44c18859050be5af314bae8f05660478af5e31d/src/api/ngx_http_lua_api.h -O --output-dir debian/
```

Comment:
Line 32 and 36, means we are pinning version right?

I think we need to think about when to update the versions. Since the renovate bot and security check would likely not scan/flag this for upgrades.

No changes needed, just some thought on this.

Reply:
Indeed. As the project is not publishing realeases nor having a clear tagging plan, we thought with Greg it was better to stick to a clear version to not introduce potential (security) issues using latest.